acceptance/build_info_test.go

```diff
@@ -31,6 +31,7 @@ func TestBuildInfo(t *testing.T) {
 	l := localcluster.Create(1, stopper)
 	l.Start()
 	defer l.Stop()
+	defer assertNoEvents(t, l)
```

Comment:
Can we make this the default so it doesn't have to be copied into every test? If events are rare enough (so we don't overflow the buffer) we could just have `l.Stop` drain the channel  and fail if there are any unexpected events in it.